src/Intl/Grapheme/Grapheme.php

```diff
@@ -105,8 +105,12 @@ public static function grapheme_strlen($s)
         return 0 === $len && '' !== $s ? null : $len;
     }
 
-    public static function grapheme_substr($s, $start, $len = 2147483647)
+    public static function grapheme_substr($s, $start, $len = null)
```

Comment:
do we even need an optional argument here ? the polyfill function always provides it.

Reply:
actually, this is required for testing.